src/utility/getClassesForSizingAndPositioning.test.js

```diff
@@ -0,0 +1,90 @@
+import {
+  getClassForAddedPackaging,
+  getClassForListedPackaging,
+  getClassForWeekdayCheckboxAlignment,
+} from "./getClassesForSizingAndPositioning";
+
+// Added products
+
+test("gives back the name of the class an image should have depending on which packaging it has", () => {
+  // GIVEN a string representing the type of packaging and a string representing the name of a product
+  const packaging = "not glass bottle";
+  const productName = "AHA + BHA";
+  // WHEN I call the function getClassForAddedPackaging with these arguments
+  const result = getClassForAddedPackaging(packaging, productName);
+  // THEN I shoud receive the correct className
+  const expected = "smaller-added-image";
+
+  expect(result).toBe(expected);
+});
+
+test("gives back an empty string if the packging isn't not glass bottle and the name isn't 100% Niacinamide Powder or 100% L-Ascorbic Acid Powder", () => {
+  // GIVEN a string representing the type of packaging and a string representing the name of a product
+  const packaging = "glass bottle";
+  const productName = "AHA + BHA";
+  // WHEN I call the function getClassForAddedPackaging with these arguments
+  const result = getClassForAddedPackaging(packaging, productName);
+  // THEN I shoud receive the correct className
+  const expected = "";
+
+  expect(result).toBe(expected);
+});
+
+// Listed Products
+
+test("gives back the name of the class for an image depending on packaging", () => {
+  // GIVEN a string representing the type of packaging and a string representing the name of a product
+  const packaging = "not glass bottle";
+  // WHEN I call the function getClassForAddedPackaging with these arguments
+  const result = getClassForListedPackaging(packaging);
+  // THEN I shoud receive the correct className
+  const expected = "smaller-image";
+
+  expect(result).toBe(expected);
+});
+
+test("gives an empty string if the packaging isn't either not glass bottle or glass container", () => {
+  // GIVEN a string representing the type of packaging and a string representing the name of a product
+  const packaging = "glass bottle";
+  // WHEN I call the function getClassForAddedPackaging with these arguments
+  const result = getClassForListedPackaging(packaging);
+  // THEN I shoud receive the correct className
+  const expected = "";
+
+  expect(result).toBe(expected);
+});
```

Comment:
These tests look very similar, consider iterating over an array with given and expected for each test